app/src/main/res/values/strings.xml

```diff
@@ -41,4 +41,12 @@
     <string name="close">Close</string>
     <string name="general_User">User</string>
     <string name="general_currency" translatable="false">€</string>
+    <string name="sign_in_registerButton_Text">Registrate now</string>
+    <string name="sign_in_editTextUsername_hint">Username</string>
+    <string name="sign_in_editTextEmail_hint" translatable="false">E-Mail</string>
+    <string name="sign_in_editTextPassword_hint">Password</string>
```

Comment:
die 3 haben wir garantiert schonmal verwendet. ist zwar nicht weiter schlimm aber unschön. Guck einfach das du in der xml auf die anderen verweist.